### PR TITLE
Broken link fixed

### PR DESCRIPTION
**Pull request by emailtovamos**
Broken link fixed: 
from 
https://kubernetes.io/docs/tasks/administer-cluster/access-cluster-api/#accessing-the-api-from-a-pod
to 
https://kubernetes.io/docs/tasks/administer-cluster/access-cluster-api/#accessing-the-api-from-within-a-pod

### CODE REVIEW

**Comment by emailtovamos** [comment]
I just signed the CLA

**Comment by emailtovamos** [comment]
I signed it

File: content/en/docs/tasks/administer-cluster/access-cluster-api.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ func main() {
 }
 ```
 
-If the application is deployed as a Pod in the cluster, please refer to the [next section](#accessing-the-api-from-a-pod).
+If the application is deployed as a Pod in the cluster, please refer to the [next section](#accessing-the-api-from-within-a-pod).
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/access-cluster-api.md]: optional rewording:
```suggestion
If the application is deployed as a Pod in the cluster, see [Accessing the API from within a Pod](#accessing-the-api-from-within-a-pod).
```

**Comment by Rajakavitha1** [comment]
Hi @emailtovamos !!! Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

Please follow instructions at https://github.com/kubernetes/community/blob/master/CLA.md#troubleshooting to resolve any issues that you face.

**Comment by emailtovamos** [comment]
Ok I have asked question regarding my CLA [here](https://jira.linuxfoundation.org/servicedesk/customer/portal/4/SUPPORT-766) as I couldn't find any issue with the way I signed it before. Hopefully they will look into it soon.

**Comment by emailtovamos** [comment]
Ok now the signing has worked.

**Pull request merged**